pkg/packaging/packaging.go

```diff
@@ -58,12 +60,6 @@ type PackageOptions struct {
 	WixSkipCleanup    bool
 	DisableService    bool
 
-	// Normally we'd download the same version we bake into the
-	// autoupdate. But occasionally, it's handy to make a package
-	// with a different version.
-	LauncherDownloadVersionOverride string
-	OsqueryDownloadVersionOverride  string
```

Comment:
It's okay to remove these, but I want to note that I have used them in the past. They might make less sense with the newer TUF updater